templates/views/account/register.pug

```diff
@@ -12,7 +12,7 @@ block content
           ul.text-left
             li <strong>You may only register one account.</strong> Additional accounts will be banned. If you misbehave using multiple accounts, you may be banned completely.
             li <strong>Your email address is the only way to regain access to your account.</strong> if you lose your password. If you lose both your password and your email, you will lose access to your account.
-            li <strong>Full FAF Social rules:</strong> <a href="https://forums.faforever.com/viewtopic.php?f=2&amp;t=581">FAF Lobby and Forum rules</a>
```

Comment:
I don' think the amp; is correct here.

Reply:
Yeah I was wondering about that too, but the new link removes that part anyway?

Reply:
change it, I was about to merge, now i can't